### DISCUSSION TITLE
How to disable ARD for multiple dimensions?

### DISCUSSION THREAD

**Discussion by patel-zeel** [thomaspinder/GPJax#158] [Q&A] [answered]
Hi, 

This might be simple, but I am unable to figure out how **not to use** ARD lengthscales for multi-input regression. I am doing some comparison study and thus this was needed. Can someone suggest a solution?

**Comment by thomaspinder** [comment, marked as answer]
Thanks for catching this. This is a bug. I've opened an issue.
